sandbox_dns_unix.go

```diff
@@ -313,8 +315,31 @@ func (sb *sandbox) rebuildDNS() error {
 	// external v6 DNS servers has to be listed in resolv.conf
 	dnsList = append(dnsList, resolvconf.GetNameservers(currRC.Content, types.IPv6)...)
 
-	// Resolver returns the options in the format resolv.conf expects
-	dnsOptionsList = append(dnsOptionsList, sb.resolver.ResolverOptions()...)
+	// If the user entered a ndots value > 0 remember that. container's resolv.conf
+	// will only have ndots:0 for embedded DNS server to get unqualified names.
+	resOptions := sb.resolver.ResolverOptions()
+	for i, option := range dnsOptionsList {
+		if strings.Contains(option, "ndots") {
+			parts := strings.Split(option, ":")
+			if len(parts) == 0 || len(parts) > 2 {
```

Comment:
can len(parts) ever be 0 for the above operation ? or should the check be for `len(parts) == 1` ?